guide/english/user-experience-design/fitts-law/index.md

```diff
@@ -3,7 +3,7 @@ title: Fitts Law
 ---
 
 ## Fitts Law
-Fitts's Law provides a model of human movement which can accurately predict the amount of time taken to move to and select a target. In human-computer interaction Fitts's Law is typically applied to movement through the graphical user interface using a cursor or other type of pointer.
+Fitts's Law provides a model of human movement which can accurately predict the amount of time taken to move to and select a target. In human-computer interaction Fitts's Law is typically applied to movement through the graphical user interface using a cursor or other type of pointer. In mobile screens this could be the distance the finger has to travel from its current position to  reach the target.
```

Comment:
```suggestion
Fitts's Law provides a model of human movement which can accurately predict the amount of time taken to move to and select a target. In human-computer interaction Fitts's Law is typically applied to movement through the graphical user interface using a cursor or other type of pointer. In mobile screens this could be the distance the finger has to travel from its current position to reach the target.
```